ci/jenkins/README.md

```diff
@@ -181,6 +181,30 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-install-ovs
 ```
 
+* rancher-e2e: e2e tests in a rancher cluster
+
+```shell
+#!/bin/bash
+DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+./ci/jenkins/test-rancher.sh --registry ${DOCKER_REGISTRY} --testcase e2e --cluster-name rancher-test
+```
+
+* rancher-conformance: conformance tests in a rancher cluster
+
+```shell
+#!/bin/bash
+DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+./ci/jenkins/test-rancher.sh --cluster-name rancher-test --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}}
```

Comment:
Better to put the rendered shell script here. Replace `{conformance_type}` with actual string value, and replace `${{DOCKER_REGISTRY}}` with `${DOCKER_REGISTRY}`. Same comment for the shell script line 205 below.